TypeScript Guides: additions and fixes (#6679)
DevExpress/devextreme-documentation: concepts/Common/TypeScript Guides/05 Scopes of Types/10 Component-Specific Types.md

---
##### jQuery

Each DevExtreme component has its own set of types. To access these types, specify `DevExpress.ui.dxComponent` where `Component` is the component name in camel case. Your IDE should give you a list of available types.

    const dateType: DevExpress.ui.dxDateBox.DateType = 'datetime';

##### Angular

> Each DevExtreme component has its own set of types. Use *aggregated exports* to import all component types with a single statement (refer to the [Aggregated Export](/concepts/Common/TypeScript%20Guides/90%20Syntax%20Elements/15%20Aggregated%20Export.md '/Documentation/Guide/Common/TypeScript_Guides/Syntax_Elements/#Aggregated_Export') help topic to learn how it works).

Reviewer: ```suggestion
Each DevExtreme component has its own set of types. Use *aggregated exports* to import all component types with a single statement (refer to the [Aggregated Export](/concepts/Common/TypeScript%20Guides/90%20Syntax%20Elements/15%20Aggregated%20Export.md '/Documentation/Guide/Common/TypeScript_Guides/Syntax_Elements/#Aggregated_Export') help topicfor more information).
```

##### Vue

> Each DevExtreme component has its own set of types. Use *aggregated exports* to import all component types with a single statement (refer to the [Aggregated Export](/concepts/Common/TypeScript%20Guides/90%20Syntax%20Elements/15%20Aggregated%20Export.md '/Documentation/Guide/Common/TypeScript_Guides/Syntax_Elements/#Aggregated_Export') help topic to learn how it works).

Reviewer: ```suggestion
Each DevExtreme component has its own set of types. Use *aggregated exports* to import all component types with a single statement (refer to the [Aggregated Export](/concepts/Common/TypeScript%20Guides/90%20Syntax%20Elements/15%20Aggregated%20Export.md '/Documentation/Guide/Common/TypeScript_Guides/Syntax_Elements/#Aggregated_Export') help topic for more information).
```

##### React

> Each DevExtreme component has its own set of types. Use *aggregated exports* to import all component types with a single statement (refer to the [Aggregated Export](/concepts/Common/TypeScript%20Guides/90%20Syntax%20Elements/15%20Aggregated%20Export.md '/Documentation/Guide/Common/TypeScript_Guides/Syntax_Elements/#Aggregated_Export') help topic to learn how it works).

Reviewer: ```suggestion
Each DevExtreme component has its own set of types. Use *aggregated exports* to import all component types with a single statement (refer to the [Aggregated Export](/concepts/Common/TypeScript%20Guides/90%20Syntax%20Elements/15%20Aggregated%20Export.md '/Documentation/Guide/Common/TypeScript_Guides/Syntax_Elements/#Aggregated_Export') help topic for more information).
```
